index.html

```diff
@@ -222,10 +227,10 @@
         </li>
         <li>If <var>wakeLockResolution</var> is <a>pending wake lock
         resolution</a> and the <a>user agent</a> does not support the <a>wake
-        lock type</a> denoted by <var>type</var>, or has otherwise determined
-        it will <a>deny wake lock</a> of this <var>type</var> for this
-        <a>browsing context</a>, set <var>wakeLockResolution</var> to
-        <a>rejected wake lock resolution</a>.
+        lock type</a> denoted by <var>type</var>, the request has been made from
+        an insecure context, or has otherwise determined it will <a>deny wake
```

Comment:
Nit: maybe, instead: 

> a context that is "Not Secure" 

That would align better with the Secure Context spec.